app/controllers/account_details.js

```diff
@@ -80,7 +80,8 @@ export default Ember.Controller.extend({
     if (bookAppointment) {
       this.transitionToRoute("request_purpose", {
         queryParams: {
-          bookAppointment: true
+          bookAppointment: true,
+          prevPath: "account_details"
```

Comment:
Now that we're using `history.back()`, do we need this?

Reply:
No we dont need any of the query params now. Will refactor whole routing for book appointment.